docroot/profiles/custom/cgov_site/modules/custom/cgov_blog/src/Plugin/Block/CgovBlogFeaturedPosts.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Drupal\cgov_blog\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+
+/*
+ * NOTE: this is a dummy plugin for front-end templating only.
+ * The innards are still being built out.
+ */
+
+/**
+ * Provides a Featured Posts Block.
+ *
+ * @Block(
+ *   id = "cgov_blog_featured_posts",
+ *   admin_label = @Translation("Blog Categories block"),
+ *   category = @Translation("Cgov Digital Platform"),
+ * )
+ */
+class CgovBlogFeaturedPosts extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $build = [
+      '#markup' => $this->t('
+        <div class="contentid-1070871 slot-item first-SI">
```

Comment:
Uhhh.... Can you either implement this or not include it?  If you are doing this so that it may be styled, then please create a feature branch for blogs.

Reply:
CgovBlogFeaturedPlugin removed.